sys/include/div.h

```diff
@@ -75,6 +100,20 @@ static inline uint64_t div_u64_by_15625(uint64_t val)
 }
 
 /**
+ * @brief Integer divide val by 625, 64 bit version
+ *
+ * @param[in]   val     dividend
+ * @return      (val / 625)
+ */
+static inline uint64_t div_u64_by_625(uint64_t val)
+{
+    if (val > 10485759996ull) {
```

Comment:
If you look at the code for the other calculation (div/15625) this kind of constant already exists. I know that this is not an "excuse" to do the same, but it was done this way to "respect" previous style.

I'll try to document both.